### PR TITLE
Slackbot handles (sorry) citations

### PR DESCRIPTION
**Pull request by PopDaph**

### CODE REVIEW

**Review by spolu — DISMISSED** [review]

**Comment by spolu** [comment]
what's the status here?

**Comment by PopDaph** [comment]
When we discussed here with @lasryaric he explained that it would be easier to test on my local when [this one](https://github.com/dust-tt/dust/pull/1798) is merged, so on my end it was paused and I started working on the reactji task, cf my goals from this morning. 
I can either configure a local slackbot tomorrow without the full local config, or see with @lasryaric if we can quickly test it on his env to merge it tomo. I'll discuss it with him tomorrow.

File: connectors/src/connectors/slack/bot.ts [hunk 1]
```diff
@@ -251,15 +253,16 @@ async function botAnswerMessage(
         break;
       }
       case "agent_generation_success": {
-        const finalAnswer = `${_removeCiteMention(
-          event.text
-        )}\n\n <${DUST_API}/w/${connector.workspaceId}/assistant/${
-          conv.conversation.sId
-        }|Continue this conversation on Dust>`;
+        const agentMessage = convRes.value.agentMessage;
```
Comment (spolu) [inline connectors/src/connectors/slack/bot.ts:256]: I'm not sure I understand this line? Where is this convRes coming from and how can it hold an up to date agentMessage?

Comment (spolu) [reply connectors/src/connectors/slack/bot.ts:256]: Oh boy I see it now.

Let's remove the 10x getConvefrsation :scream:

Comment (spolu) [reply connectors/src/connectors/slack/bot.ts:256]: Probably another PR :/

Comment (spolu) [reply connectors/src/connectors/slack/bot.ts:256]: Crated https://github.com/orgs/dust-tt/projects/3/views/1?pane=issue&itemId=40134811

**Comment by spolu** [comment]
Currently blocked on https://github.com/orgs/dust-tt/projects/3/views/1?pane=issue&itemId=40134811

**Comment by spolu** [comment]
@PopDaph r? PTAL I rebased on top of main, testing now
@lasryaric r? as well :pray:

**Review by lasryaric — DISMISSED** [review]
LGTM

**Comment by spolu** [comment]
Working on last tweaks, context here: https://dust4ai.slack.com/archives/C050DRFBYGK/p1696243007947219

**Comment by spolu** [comment]
All done. post-merge review welcome.

@lasryaric I bumped to 1500ms the interval as it looks like we have a cap on the number of updates for large messages. Won't fix all messages but push a bit further.

As a work around I also inject the final link always with ... while the message is not finished

**Pull request merged**